src/mono_camera.cpp

```diff
@@ -162,4 +167,18 @@ void MonoCamera::updateCameraInfo(const avt_vimba_camera::AvtVimbaCameraConfig&
   info_man_->setCameraInfo(ci);
 }
 
+//time_compare compares the system time with time stamp and if its greater than 3 sec it assigns OFF status
+void MonoCamera::updateCameraStatus()
+{
+    ros::Duration img_time_difference=ros::Time::now()-last_time_;
+    ros::Duration three_seconds(2.0);
```

Comment:
No need to declare a variable here. Just use a Duration in your comparison like:
```img_time_difference > ros::Duration(2.0)```

Reply:
Please add comments on why you chose 2 seconds.